docker/develop/docker-compose.yml

```diff
@@ -21,8 +22,18 @@ services:
       - dia_data_bundle:/diaspora/vendor/bundle
     ports:
       - ${DIASPORA_DOCKER_PORT:-3000}:3000
+    environment:
+      - ENVIRONMENT_REDIS=redis://redis
+      - SERVER_LISTEN=tcp://0.0.0.0:3000
     depends_on:
       - "${DIASPORA_DOCKER_DB}"
+      - redis
+
+  redis:
+    image: redis:7
+    command: redis-server --save 60 1 --loglevel warning
+    volumes:
+      - redis_data:/data
```

Comment:
Technically this isn't needed, as we don't care too much if the data in Redis doesn't survive a restart for dev - but it doesn't matter. :)

Reply:
Yes and no, I thought about that. You are right, with the `single_process_mode` we didn't keep anything before, but when doing federation tests (where it also was required to disable `single_process_mode` before) or something, it might be useful to keep the state (and retries or scheduled jobs), so I thought it doesn't hurt.

Reply:
Yeah, that's reasonable!